packages/service-library/tests/test_rabbitmq.py

```diff
@@ -86,10 +87,36 @@ def mocked_message_parser(mocker: MockerFixture) -> mock.AsyncMock:
     return mocker.AsyncMock(return_value=True)
 
 
+@dataclass(frozen=True)
+class PytestRabbitMessage:
+    message: str
+    topic: str
+
+    def routing_key(self) -> str:
+        return self.topic
+
+    def body(self) -> bytes:
+        return self.message.encode()
+
+
+@pytest.fixture
+def random_rabbit_message(
+    faker: Faker,
+) -> Callable[..., PytestRabbitMessage]:
+    def _creator(**kwargs: dict[str, Any] | None) -> PytestRabbitMessage:
+        msg_config = {"message": faker.text(), "topic": None}
+        if kwargs:
+            msg_config |= kwargs
```

Comment:
therefore this would just ``msg_config ={"message": faker.text(), "topic": None, **kwargs} ``

or simply  ``return PytestRabbitMessage("message": faker.text(), "topic": None, **kwargs)``

Reply:
thanks!